guides/concepts/list-of-operations.md

```diff
@@ -131,75 +133,122 @@ If you want to delete an existing offer set Offer ID to existing offer ID and se
-|MANAGE_OFFER_SELL_NOT_AUTHORIZED| -4| The account creating the offer is not authorized to sell this asset.|
-|MANAGE_OFFER_BUY_NOT_AUTHORIZED| -5| The account creating the offer is not authorized to buy this asset.|
-|MANAGE_OFFER_LINE_FULL| -6| The account creating the offer does not have sufficient limits to receive `buying` and still satisfy its buying liabilities.|
-|MANAGE_OFFER_UNDERFUNDED| -7| The account creating the offer does not have sufficient limits to send `selling` and still satisfy its selling liabilities. Note that if selling XLM then the account must additionally maintain its minimum XLM reserve, which is calculated assuming this offer will not completely execute immediately.|
-|MANAGE_OFFER_CROSS_SELF| -8| The account has opposite offer of equal or lesser price active, so the account creating this offer would immediately cross itself.|
-|MANAGE_OFFER_SELL_NO_ISSUER| -9| The issuer of selling asset does not exist.|
-|MANAGE_OFFER_BUY_NO_ISSUER| -10| The issuer of buying asset does not exist.|
-|MANAGE_OFFER_NOT_FOUND| -11| An offer with that `offerID` cannot be found.|
-|MANAGE_OFFER_LOW_RESERVE| -12| The account creating this offer does not have enough XLM to satisfy the minimum XLM reserve increase caused by adding a subentry and still satisfy its XLM selling liabilities. For every offer an account creates, the minimum amount of XLM that account must hold will increase.|
-
-## Create Passive Offer
-[JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.createPassiveOffer) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/CreatePassiveOfferOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/build#ManageOfferBuilder)
-
-A passive offer is an offer that does not act on and take a reverse offer of equal price. Instead, they only take offers
-of lesser price. For example, if an offer exists to buy 5 BTC for 30 XLM, and you make a passive offer to buy 30 XLM for 5 BTC,
-your passive offer *does not* take the first offer.
-
-Note that regular offers made later than your passive offer can act on and take your passive offer, even if the regular
-offer is of the same price as your passive offer.
-
-Passive offers allow market makers to have zero spread. If you want to trade EUR for USD at 1:1 price and USD for EUR also
- at 1:1, you can create two passive offers so the two offers don't immediately act on each other.
-
-Once the passive offer is created, you can manage it like any other offer using the [manage offer](#manage-offer) operation.
+| ----- | ---- | ------ |
+| MANAGE_BUY_OFFER_MALFORMED | -1 | The input is incorrect and would result in an invalid offer. |
+| MANAGE_BUY_OFFER_BUY_NO_TRUST | -2 | The account creating the offer does not have a trustline for the asset it is selling. |
+| MANAGE_BUY_OFFER_BUY_NO_TRUST | -3 | The account creating the offer does not have a trustline for the asset it is buying. |
```

Comment:
These two error codes duplicated. I recommend comparing against [the XDR file](https://github.com/stellar/stellar-core/blob/df28f6481c211f0812cb2e5e2132df0d461f13a3/src/xdr/Stellar-transaction.x#L558) for correctness. Here you want
```
 MANAGE_BUY_OFFER_SELL_NO_TRUST = -2, // no trust line for what we're selling
MANAGE_BUY_OFFER_BUY_NO_TRUST = -3, // no trust line for what we're buying
```

Reply:
Yeah this came about via a find and replace, good catch.